src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -17,6 +17,16 @@
             <wpf:BoolToVisibilityConverter TrueValue="Collapsed" FalseValue="Visible" />
         </wpf:AggregatorValueConverter>
 
+        <ContextMenu x:Key="RowMenu" DataContext="{Binding PlacementTarget.DataContext, RelativeSource={RelativeSource Self}}">
+            <MenuItem Header="Delete" 
```

Comment:
`Delete` sounds a bit final. Could it mean the issue is deleted on the server? 
`Remove` sounds less final and less likely to be misinterpreted.